.github/workflows/ci.yml

```diff
@@ -147,6 +153,13 @@ jobs:
         with:
           node-version: ${{ matrix.language_version }}
 
+      - if: matrix.language == 'python'
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.language_version }}
+      - if: matrix.language == 'python'
+        run: pip install build hatch pytest
```

Comment:
> outside view tells us that uv will not be the last attempt to fix Python packaging

Heh!

I went with `hatch` for the build, purely because it's what's recommended by default in the packaging tutorial. Our building is extremely simple, and these are only build time deps (the resulting package doesn't depend on any of them) so it's going to be easy to change in future if we decide we need to.